v1/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/appoptics/appoptics-apm-go/v1
+
+go 1.12
+
+require (
+	github.com/cespare/xxhash v1.1.0 // indirect
```

Comment:
what does the comment `// indirect` mean in this context?

Reply:
It means:

> The indirect comment indicates a dependency is not used directly by this module, only indirectly by other module dependencies.

From: https://blog.golang.org/using-go-modules

Reply:
ok, do yo know what dependency uses this module? If so, it might be helpful to have a comment like `// used indirectly by module x`. Or if that is unknown you could say `// used indirectly by dependency module`

Reply:
The `go.mod` file is generated and updated by the go toolchains automatically and they don't expect us to modify it.

Reply:
As there might be multiple modules depending on the same item and it's hard to maintain the comment. I guess that's why they didn't put the module names here.

Reply:
ah ok, didn't know I was commenting on an auto-generated piece of code :D in that case no changes needed. thanks.